flight/targets/brainre1/re1fpga/fpga_drv.c

```diff
@@ -618,6 +565,90 @@ void PIOS_RE1FPGA_Set3DConfig(enum pios_video_3d_mode mode, uint8_t x_shift_righ
 	PIOS_RE1FPGA_WriteReg(RE1FPGA_REG_XCFG2, cfg, 0xFF);
 }
 
+// **********************************************************************************
+// PIOS WS2811 support functions
+
+struct ws2811_pixel_data_s {
+	uint8_t g;
+	uint8_t r;
+	uint8_t b;
+};
+
+struct ws2811_dev_s {
+#define WS2811_MAGIC 0x31313832		/* '2811' */
+	uint32_t magic;
+	uint16_t max_leds;
+	uint16_t max_idx;
+	struct ws2811_pixel_data_s pixel_data[0];
+};
+
+int PIOS_WS2811_init(ws2811_dev_t *dev_out, const struct pios_ws2811_cfg *cfg,
```

Comment:
I like this approach of replicating the signatures from the other driver to avoid conditionals in the users.